requirements.txt

```diff
@@ -1,6 +1,5 @@
 PyAudio==0.2.8
 SpeechRecognition==3.8.1
 click==6.7
-numpy==1.16.2
```

Comment:
Are you sure? I believe numpy is still required. See scripts/respeaker_node.py line 11